ppr-ui/src/components/mhrRegistration/ReviewConfirm/YourHomeReview.vue

```diff
@@ -16,136 +16,137 @@
         </span>
       </section>
 
-      <!-- Manufacturer Make Model -->
-      <section class="py-6">
-        <v-row no-gutters class="px-6">
-          <v-col cols="3">
-            <h3>Manufacturer's Name</h3>
-          </v-col>
-          <v-col cols="9">
-            <p>{{ getMhrRegistrationHomeDescription.manufacturer || '(Not Entered)' }}</p>
-          </v-col>
-        </v-row>
-        <v-row no-gutters class="pt-3 px-6">
-          <v-col cols="3">
-            <h3>Year of Manufacture</h3>
-          </v-col>
-          <v-col cols="9">
-            <p v-if="getMhrRegistrationHomeDescription.baseInformation.year">
-              {{ getMhrRegistrationHomeDescription.baseInformation.circa
-              ? 'Circa ' + getMhrRegistrationHomeDescription.baseInformation.year
-              : getMhrRegistrationHomeDescription.baseInformation.year
-              }}
-            </p>
-            <p v-else>(Not Entered)</p>
-          </v-col>
-        </v-row>
-        <v-row no-gutters class="pt-3 px-6">
-          <v-col cols="3">
-            <h3>Make</h3>
-          </v-col>
-          <v-col cols="9">
-            <p>{{ getMhrRegistrationHomeDescription.baseInformation.make || '(Not Entered)'  }}</p>
-          </v-col>
-        </v-row>
-        <v-row no-gutters class="pt-3 px-6">
-          <v-col cols="3">
-            <h3>Model</h3>
-          </v-col>
-          <v-col cols="9">
-            <p>{{ getMhrRegistrationHomeDescription.baseInformation.model || '(Not Entered)'  }}</p>
-          </v-col>
-        </v-row>
-      </section>
-
-      <v-divider class="mx-4"/>
-
-      <!-- CSA Review -->
-      <template v-if="isCSA || isEngineerInspection">
-        <template v-if="isCSA">
-          <v-row no-gutters class="pa-6">
-            <v-col cols="3" class="pt-1">
-              <h3>CSA Number</h3>
+      <template v-if="hasData">
+        <!-- Manufacturer Make Model -->
+        <section class="py-6">
+          <v-row no-gutters class="px-6">
+            <v-col cols="3">
+              <h3>Manufacturer's Name</h3>
```

Comment:
Not sure why git is showing so many changes, I just added a template tag and indented everything by 1.

Reply:
It's probably the spacing, since you added <template> on line 19, so everything else became nested.

Reply:
Yeah your right. If you hide whitespace, it much easier to review!